src/skins/vector/css/matrix-react-sdk/structures/_MatrixChat.scss

```diff
@@ -100,6 +100,11 @@ limitations under the License.
      * flex itself.
```

Comment:
Right, I even commented this: "Height has to be auto here for RoomView to correctly fit when the Toolbar is shown".  This is why it's not set to 100%.  Unless the phase of the moon has changed since I wrote this, I'd expect it still to be true.

Luke: have you tested this with the MatrixToolbar visible across the top of the page (e.g. showing 'you're a guest' or 'new version' text)?